### DISCUSSION TITLE
[indexer] Get ready for nearcore 1.22.0

### DISCUSSION THREAD

**Discussion by khorolets** [near/nearcore#5041] [Node Public Interfaces]
`nearcore` is planned to be updated to version 1.22.0:
- *testnet* on ~~Oct 18, 2021~~ Oct 19, 2021 (it's recommended to upgrade your nodes before Oct 21, 2021)
- *mainnet* on Nov 15, 2021 (recommended to upgrade your nodes before Nov 17, 2021)

Protocol upgrade is expected which will enable sharding. Please read about hardware requirement changes here https://mailchi.mp/82ea6151042f/preparing-for-the-upcomingnearcore-v1220-release

## Quick info

**Commit hash** [cd5b52a160f36eabd95ad1637aebc9e6ee041a2d](https://github.com/near/nearcore/commit/cd5b52a160f36eabd95ad1637aebc9e6ee041a2d) (this is [1.22.0-rc.5](https://github.com/near/nearcore/releases/tag/1.22.0-rc.5) release, while 1.22.0 release will be tagged on Nov 15, 2021)
**Expected issues** none
**Reference** have a look at how [we made it on NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer/pull/179/files)

Feel free to leave your questions in the comments below.

**Comment by khorolets** [comment]
Heads up. I've updated the recommended commit based on @bowenwang1996 recommendation from here https://github.com/near/nearcore/issues/5049#issuecomment-948871714

**Comment by frol** [comment]
Heads up, the recommended commit hash is updated to 1.22.0-rc.5 release.
